docutron/src/components/Page.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import { getNextStory, getPreviousStory } from 'docutron';
```

Comment:
It seems a bit odd for this module to depend from itself. Should we instead import by traversing to the relative path?